### ISSUE TITLE
TypeError:mul():argument 'other' (position 1) must be Tensor,not list

### ISSUE THREAD

**Issue by ChiYing212** [Nealcly/LAN#6]
when I run the main.py, the above problems occurs in line 213
(line 213 is "mask[idx,:seqlen] = torch.Tensor([1]*seqlen)")

**Comment by Nealcly** [comment]
May I ask which Pytorch version you used?

**Comment by ChiYing212** [comment]
0.3.1

**Comment by satori555** [comment]
> when I run the main.py, the above problems occurs in line 213
> (line 213 is "mask[idx,:seqlen] = torch.Tensor([1]*seqlen)")

Pytorch 0.3.1 shoule be ok, for newer version, just modify seqlen as seqlen.item()

**Comment by wshzd** [comment]
torch                0.3.0b0+591e73e
that is my torch version above,l can run correctly.

**Comment by chiyuzhang94** [comment]
Hi, I am also trying to use the tool. But I am wondering where the dataset is. I saw you run this model and posted issues. What dataset did you use? Could you please share with me?

**Comment by yhjiujiu** [comment]
mask[idx,:seqlen] = torch.Tensor(torch.tensor([1])*seqlen)"

**Comment by popkarthb** [comment]
Check your input data parameter.
Make sure it is in 123123 not in [123123]
